pkg/storkctl/snapshotschedule.go

```diff
@@ -146,6 +147,72 @@ func newGetSnapshotScheduleCommand(cmdFactory Factory, ioStreams genericclioptio
 	return getSnapshotScheduleCommand
 }
 
+// newUpdateSnapShotScheduleCommand implements the `storkctl update` command and it's utilities.
+func newUpdateSnapShotScheduleCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	var newPVCName string
+	updateSnapshotScheduleCommand := &cobra.Command{
+		Use:     snapshotScheduleSubcommand,
+		Aliases: snapshotScheduleAliases,
+		Short:   "Update snapshot schedules",
+		Run: func(c *cobra.Command, args []string) {
+			var msg, snapshotSchedule string
+			var found = false
+
+			if len(args) == 0 {
+				util.CheckErr(fmt.Errorf("argument needs to be provided for snapshot schedule name"))
+				return
+			}
+			snapshotSchedule = args[0]
+			if len(snapshotSchedule) == 0 {
+				handleEmptyList(ioStreams.Out)
+				return
+			}
+
+			// Check if any of the flags have been provided.
+			c.Flags().VisitAll(func(flag *pflag.Flag) {
+				name := flag.Name
+				if c.Flags().Changed(name) {
+					found = true
+					return
+				}
+			})
+
+			if len(newPVCName) != 0 {
+				if err := updateSnapshotSchedule(snapshotSchedule, cmdFactory.GetNamespace(), newPVCName); err != nil {
+					util.CheckErr(err)
+					return
+				}
+			}
+
+			// Output the result.
+			if !found {
+				msg = "no valid flags provided"
+			} else {
+				msg = fmt.Sprintf("VolumeSnapshotSchedule %v updated successfully", snapshotSchedule)
+			}
+			printMsg(msg, ioStreams.Out)
+		},
+	}
+	updateSnapshotScheduleCommand.Flags().StringVar(&newPVCName, "new-pvc-name", "", "Specify new PVC name of the volume snapshot schedule")
```

Comment:
nit: Specify the PVC name to be updated in the volumesnapshotschedule

Reply:
Made the change.